examples/image_reconstruction.ipynb

```diff
@@ -0,0 +1,705 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "CMdgh_Jmw_Pu"
+   },
+   "source": [
+    "<div class=\"alert alert-success\">\n",
+    "    <h1 align=\"center\">Image reconstruction</h1>\n",
+    "    <h1 align=\"center\">Using the Poutyne framework</h1>\n",
+    "    <h3 align=\"center\"><a href=\"https://poutyne.org/\"> Click on this tab for more information</a></h3>\n",
+    "</div>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "Zk7-1o9lw_Py"
+   },
+   "source": [
+    "In this example, we train a simple convolutional autoencoder (Conv-AE) on MNIST dataset to learn image reconstruction. The Conv-AE is composed of two parts: an encoder and a decoder. The encoder encodes the input images to extract compact image features. The extracted feature (here feature map, as all layers are convolutional) goes through the decoder to reconstruct the input image."
```

Comment:
"[...] (here feature map, as all layers are convolutional) [...]" do you refer to the figure below or in all the example?
Maybe link to the figure below?